### ISSUE TITLE
Clarify multi-window usage

### ISSUE THREAD

**Issue by arousle** [NDevTK/AutoPause#22]
Hey! Thanks for the neat extension. 

I think I may either not fully understand how to use it, though. Or maybe what I'm trying to achieve isn't possible?

Here's what I'm trying to achieve:

- 2 Firefox windows are open: Window A and Window B.
- Window A has two media tabs loaded: Tab A1 and Tab A2. I want AutoPause to make sure that only one of these is currently playing.
  - (Both of them are [Picarto ](https://picarto.tv/) streams in my particular use case, if it matters.)
- _While Window A is showing_, I want to be able to switch to Window B without the media currently playing in Window A to pause.
  - In other words, I want to see both windows side-by-side, and have interaction with Window B not interfere with the playing of media in Window A.

Is this possible? When I have the "Pause media on tab change (...)" setting enabled and I switch from Window A to Window B, the media in Window A will pause playback, which I _don't_ want. With the setting _disabled_, playback in Tab A1 will not pause when I switch  to Tab A2 in Window A (as far as I can tell).

**Comment by NDevTK** [comment]
Are you asking for an option to only pause inactive tabs?
You can current ignore tabs using the shortcut.

**Comment by arousle** [comment]
Thanks for the response, but I don't think that's what I want, no.

My scenario is that I have one browser window open, which I use for playing video on a separate screen. Let's call this the "video window". It may have any number of tabs with media content open, and I want to use AutoPause to pause any playback in non-active tabs _for that window_.

I also have one or more other browser windows open on other screens, again with any number of tabs per window. These I use for regular browsing. I want to use these other browser windows without interfering with the "video window".

Currently, with the "Pause media on tab change (...)" setting enabled, video playback of the active tab in the "video window" stops when foregrounding one of the _other_ windows. I don't want that to happen, but I _do_ want tabs in the "video window" to pause/resume when switching tabs _in the "video window"_.

**Comment by NDevTK** [comment]
So do you want a window to be marked for AutoPause use and ignore every other window?

**Comment by arousle** [comment]
> So do you want a window to be marked for AutoPause use and ignore every other window?

I think that would work, yes!

**Comment by NDevTK** [comment]
Should be fixed in 2.7.4 please confirm.

**Comment by arousle** [comment]
Yes, I can get it to work now. Thanks for the quick response!
